code/modules/reagents/chemistry/recipes/slime_extracts.dm

```diff
@@ -70,6 +70,14 @@
 	required_other = TRUE
 	required_container = /obj/item/slime_extract/green
 
+/datum/chemical_reaction/slime/slimeethereal
+	name = "Ethereal Mutation Toxin"
+	id = "etherealarerealmuttoxin"
+	results = list(/datum/reagent/mutationtoxin/ethereal = 1)
+	required_reagents = list(/datum/reagent/consumable/liquidelectricity = 1)
```

Comment:
Maybe teslium instead?

Reply:
yeah that sounds better ill change it in a bit

Reply:
was just thinking that liquid electricty was available already from high power energy bar and its also the blood of ethereals